test/index.js

```diff
@@ -54,11 +54,16 @@ test('#load', function(t){
 });
 
 test('#pack', function(t){
-  t.plan(1);
+  t.plan(2);
 
-  var crx = newCrx();
+  var crx2 = newCrx();
+  crx2.pack().then(function(packageData){
+    t.ok(packageData instanceof Buffer);
+  })
+  .catch(t.error.bind(t));
 
-  crx.pack().then(function(packageData){
+  var crx3 = newCrx({version: 3});
```

Comment:
The rest of the tests should be updated to test both versions too.

Reply:
@arkon which ones? Because most of them shouldn't differ between versions. I guess end-to-end one could test both versions.

Reply:
@arkon nvm, i'll rewrite test to run all tests for each format version.

Reply:
Done.